portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,46 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
```

Comment:
Imports are conventionally kept in alphabetical order.

On a related note, you can refer to the open-sourced Google Java language style guide[1] for more details.

[1] https://google.github.io/styleguide/javaguide.html